.github/workflows/devnet_docker.yml

```diff
@@ -53,12 +53,13 @@ jobs:
           labels: ${{ steps.meta.outputs.labels }}
           build-args: |
             ADMIN=${{ vars.DEVNET_ADMIN }}
+            DEPLOYER_PRIVATE_KEY=${{ vars.DEVNET_ADMIN_PRIVATE_KEY }}
             FACTORY_MIN_CURVE_FEE=0
             FACTORY_MIN_FLAT_FEE=0
             FACTORY_MIN_GOVERNANCE_LP_FEE=0
             FACTORY_MIN_GOVERNANCE_ZOMBIE_FEE=0
-            FACTORY_MAX_CURVE_FEE=1000000000000000000
-            FACTORY_MAX_FLAT_FEE=1000000000000000000
-            FACTORY_MAX_GOVERNANCE_LP_FEE=1000000000000000000
-            FACTORY_MAX_GOVERNANCE_ZOMBIE_FEE=1000000000000000000
-            FACTORY_MIN_POSITION_DURATION=86400
+            FACTORY_MAX_CURVE_FEE=1
```

Comment:
Why are these all 1 now?

Reply:
NVM, I see that it's because they are parsed differently. This indirection is a bit subtle, so I think some inline comments like `# 1e18` would go a long way towards making this more readable.

Reply:
added, good call!